application/config/github/organizations/webhooks.js

```diff
@@ -118,7 +118,11 @@ module.exports = {
                     required     : false,
                     description  : 'Determines what events the hook is triggered for.',
                     location     : 'body',
-                    type         : 'array[string]'
+                    type         : 'array',
+                    param        : {
+                        type : 'string',
+                        name : 'event'
```

Comment:
currently it does nothing in the context of an array except that it's used by the <ParamRow> as a prop